src/Illuminate/Database/Migrations/Migrator.php

```diff
@@ -230,7 +231,7 @@ public function rollback($paths = [], array $options = [])
         // We want to pull in the last batch of migrations that ran on the previous
         // migration operation. We'll then reverse those migrations and run each
         // of them "down" to reverse the last migration "operation" which ran.
-        $migrations = $this->getMigrationsForRollback($options);
+        $migrations = isset($options['selected']) && count($options['selected']) > 0 ? $options['selected'] : $this->getMigrationsForRollback($options);
```

Comment:
Function call can be avoided like this:
```suggestion
        $migrations = isset($options['selected']) && $options['selected'] !== [] ? $options['selected'] : $this->getMigrationsForRollback($options);
```